src/app/components/Account/ConsensusAccountDetailsView.tsx

```diff
@@ -0,0 +1,103 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { CardEmptyState } from '../CardEmptyState'
+import { Account } from '../../../oasis-nexus/api'
+import { useScreenSize } from '../../hooks/useScreensize'
+import { TextSkeleton } from '../Skeleton'
+import { StyledDescriptionList, StyledListTitleWithAvatar } from '../StyledDescriptionList'
+import { DashboardLink } from '../../pages/ParatimeDashboardPage/DashboardLink'
+import Box from '@mui/material/Box'
+import { styled } from '@mui/material/styles'
+import { AccountAvatar } from '../AccountAvatar'
+import { AccountSizeBadge } from '../AccountSizeBadge'
+import { AccountLink } from './AccountLink'
+import { CopyToClipboard } from '../CopyToClipboard'
+import { getPreciseNumberFormat } from '../../../locales/getPreciseNumberFormat'
+
+export const StyledListTitle = styled('dt')(({ theme }) => ({
+  marginLeft: theme.spacing(4),
+}))
+
+type ConsensusAccountDetailsViewProps = {
+  account?: Account
+  isError?: boolean
+  isLoading?: boolean
+  showLayer?: boolean
+  standalone?: boolean
+}
+
+export const ConsensusAccountDetailsView: FC<ConsensusAccountDetailsViewProps> = ({
+  account,
+  isError,
+  isLoading,
+  showLayer,
+  standalone,
+}) => {
+  const { t } = useTranslation()
+  const { isMobile } = useScreenSize()
+
+  if (!account || isLoading) return <TextSkeleton numberOfRows={7} />
+  if (isError || (!isLoading && !account)) return <CardEmptyState label={t('account.cantLoadDetails')} />
```

Comment:
Why is this not `if (isLoading)` and `if (isError)`?

Reply:
> Why is this not `if (isLoading)` and `if (isError)`?

Updated to `if (isLoading)` and `if (isError || !account)`

Testing for account in the latter is required to handle the (hypothetical) situation when `account` is undefined.

Before #1422, this was handled by adding ` if (!account) return null`, after the loading/error logic, but @lubej didn't like returning null instead of a proper explanation, so it has been changed.

Reply:
This sounds like inaccurate types. Account should be defined after loading without an error

Reply:
useQuery types for comparison https://github.com/TanStack/query/blob/59eef10c1afa5b16ffcac4cd805d86975640f927/packages/query-core/src/types.ts#L675-L718

Reply:
OK, the type definitions basically confirm that the data will be defined except when loading or on error. 
So, adding it as a condition here doesn't cause any change in behavior, but it satisfies the TS compiler.

So I think we are OK.